hw/top_earlgrey/syn/chip_earlgrey_asic.ot.sdc

```diff
@@ -0,0 +1,1570 @@
+#    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+#    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+#               [get_nets -segments -of_objects \
+#                 [get_pins top_earlgrey/u_spi_device/u_p2s/data_valid_i]]]
+#set_false_path -hold -from [get_clocks SPI_DEV_FAST_PASS_IN_CLK] \
+#    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+#    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+#               [get_nets -segments -of_objects \
+#                 [get_pins top_earlgrey/u_spi_device/u_p2s/data_i*]]]
+
+# This path is from locality logic that is on the *_IN_CLK domain and selects
+# between fixed values or the return-by-hw register value. The flopped bits
+# settle in the middle of the command/address phase, many cycles before the
+# data phase.
+set_false_path -hold -from [get_clocks SPI_DEV_FAST_PASS_IN_CLK] \
+    -to [get_ports ${SPI_DEV_DATA_PORTS}] \
+    -through [get_pins -leaf -filter "@pin_direction == out" -of_objects \
+               [get_nets -segments -of_objects \
+                 [get_pins top_earlgrey/u_spi_device/u_spi_tpm/miso_o]]]
+
+# Remove scan paths for timing analysis
+set_clock_sense -stop_propagation -clock SPI_DEV_FAST_PASS_CLK [get_pins -leaf -filter "@pin_direction == in" -of_objects [get_nets -segments -of_objects [get_pins u_ast/u_scan_clk/in_i*]]]
+set_false_path -from [get_clocks SPI_DEV_FAST_PASS_CSB_CLK] \
+    -through [get_pins -leaf -filter "@pin_direction == in" -of_objects \
+        [get_nets -segments -of_objects \
+            [get_pins u_ast/u_scan_rst_n/in_i*]]]
+
+# Alex Williams SPI_TPM constraints
+set_false_path -from [get_clocks SPI_TPM_CLK] -through [get_pins -leaf -filter "@pin_direction == out" -of_objects [get_nets -segments -of_objects  [get_pins top_earlgrey/u_spi_device/u_s2p/data_valid_o]]]                               
+set_false_path -from [get_clocks SPI_TPM_CLK] -through [get_pins -leaf -filter "@pin_direction == out" -of_objects [get_nets -segments -of_objects [get_pins top_earlgrey/u_spi_device/u_s2p/data_o*]]]
+ set_false_path -from [get_clocks SPI_TPM_CLK] -to [get_pins -leaf -filter "@pin_direction == in" -of_objects [get_cells -filter "@is_sequential" top_earlgrey/u_spi_device/u_passthrough/*]]
+
+# TBD need Alex approve
+set_multicycle_path -setup 2 -from [get_ports ${TPM_CSB_PORT}]     -to [get_clocks SPI_TPM_CLK]
+set_multicycle_path -hold -end 1 -from [get_ports ${TPM_CSB_PORT}] -to [get_clocks SPI_TPM_CLK]
+
+
+set_false_path -from [get_clocks SPI_DEV_CLK] -through [get_cells top_earlgrey/u_sysrst_ctrl_aon/*]
+set_false_path -from [get_clocks SPI_DEV_HC_CLK] -through [get_cells top_earlgrey/u_sysrst_ctrl_aon/*]
+set_false_path -from [get_clocks SPI_HOST_CLK] -through [get_cells top_earlgrey/u_sysrst_ctrl_aon/*]
+set_false_path -from [get_clocks SPI_DEV_FAST_PASS_CLK] -through [get_cells top_earlgrey/u_sysrst_ctrl_aon/*] ; #TBD should be approved
```

Comment:
Has this happened?

Reply:
The comment has been updated to say that this constraint got added based on interaction between Leonid and @a-will . This looks okay to me.